CHANGELOG.md

```diff
@@ -1,14 +1,17 @@
 # Changelog
-* Added *-c, --command* option in *generate-docs* to generate specific command from integration.
-* Fixed an issue in getting a README/CHANGELOG files from git and loading it.
+
+
+#### 0.5.2
```

Comment:
Why `0.5.2` and not `0.5.1`?

Reply:
The 0.5.1 was pre-released, it has also the same changelog, but the latest that was actually released was 0.5.2